### PR TITLE
Text edits from Chad

### PR DESCRIPTION
**Pull request by lindsayplatt**
This includes the straight-forward text edits from Chad. More needed and described in [this MS Teams thread](https://teams.microsoft.com/l/message/19:8a8e6061dec241de887ebf2f2b1d6e41@thread.skype/1597434180416?tenantId=0693b5ba-4b18-4d7b-9341-f32f400a5494&groupId=80983861-ab36-4fdc-a217-17742ba5ee3d&parentMessageId=1597434180416&teamName=GS-WMA-IIDD-Makerspace&channelName=Project%20-%20Gages%20Through%20Ages&createdTime=1597434180416).

### CODE REVIEW

File: src/components/GagesBarChartAnimation.vue [hunk 1]
```diff
@@ -4859,7 +4866,7 @@
               <text
                 class="cls-61"
                 transform="translate(701.7 1184.5)"
-              >What was the storm surge?</text>
```
Comment (jread-usgs) [inline src/components/GagesBarChartAnimation.vue:4862]: woa, how did this old surge text get in here? I thought I changed this awhile ago

Comment (lindsayplatt) [reply src/components/GagesBarChartAnimation.vue:4862]: I thought we had changed it, too ?

Comment (lindsayplatt) [reply src/components/GagesBarChartAnimation.vue:4862]: Might be from the infographic changes being pretty manual

Comment (jread-usgs) [reply src/components/GagesBarChartAnimation.vue:4862]: This one needs a change too `What rivers are at risk?` was something about roads instead

**Pull request merged**